docs/ldml/tr35-numbers.md

```diff
@@ -578,6 +578,20 @@ If there is an explicit negative subpattern, it serves only to specify the negat
 
 > **Note:** The thousands separator and decimal separator in patterns are always ASCII ',' and '.'. They are substituted by the code with the correct local values according to other fields in CLDR. The same is true of the - (ASCII minus sign) and other special characters listed above.
 
+A currency decimal pattern normally contains a currency symbol placeholder (one to four ¤ characters). The currency symbol placeholder may occur before the first digit, after the last digit symbol, or where the decimal symbol would otherwise be placed.
```

Comment:
one to five, not one to four.  According to the HTML version of the spec, 3 = display name, 4 = undefined, 5 = narrow.  I fixed the Markdown in #1139 to reflect what the HTML said.

Reply:
fixed